chall05/szavarce.c

```diff
@@ -0,0 +1,42 @@
+int ft_found(char *find, char *in_str)
+{
+    int pos;
+    int posb;
+    int ret;
+
+    pos = 0;
+    posb = 0;
+    ret = 1;
+    while (in_str[pos] != '\0')
+    {
+        posb = 0;
+        ret = 1;
+        while (find[posb])
+        {
+            if (find[posb] != in_str[pos + posb])
+                ret = 0;
+            posb++;
+        }
```

Comment:
Here you will end up reading outside of `in_str` reserved memory. You should check for `in_str`'s end too! You will crash in my tests.